user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_editing_connection_string.md

```diff
@@ -21,7 +21,14 @@ To do so:
    - *Update All Connections To This Agent*: Select this option if you want to update all connection strings to the DataMiner Agent specified in *To*.
    - *New Connection String*: The IP address to be used when connecting to the DataMiner Agent specified in *To*. Syntax example: `http://[IP address to connect]:8004/SLNetService`.
 
-1. Click *OK* to save the configuration, and then click *Done*.
+     > [!NOTE]
+     > If you want to configure gRPC connections, you can do so by adding the IP address followed by `/APIGateway`, e.g. `https://10.4.2.92:APIGateway`. You will need to do so for each DMA in the system.
```

Comment:
I would reword the note as follows:
> From DataMiner 10.3.2 onwards: to configure a gRPC connection string, provide the full URL, for example `https://10.4.2.92/APIGateway`.

Reply:
Is it OK the way I've reworded it now, or are further changes needed?